api/app/src/Repository/PreRegistrationRepository.php

```diff
@@ -92,15 +102,23 @@ public function getNewClientsForExistingDeputiesArray(): array
             pr.client_address_5   AS "ClientAddress5",
             pr.client_postcode    AS "ClientPostcode"
         FROM pre_registration pr
-        LEFT JOIN dd_user u ON pr.deputy_uid = u.deputy_uid::varchar(30)
-        LEFT JOIN deputy_case dc ON u.id = dc.user_id
-        LEFT JOIN client c ON dc.client_id = c.id
-        WHERE c.case_number != pr.client_case_number
+        WHERE
+            -- only entries in pre_registration table with an entry in the dd_user table
+            (SELECT COUNT(1) FROM dd_user u WHERE pr.deputy_uid = u.deputy_uid::varchar(30) LIMIT 1) > 0
```

Comment:
not sure what the limit 1 is for here

Reply:
I suppose my thought was it would reduce the time for the query, as it can complete as soon as it's found a match. I don't think deputy UID has a unique constraint on it, so the query could potentially carry on looking for more matching rows. But perhaps I've got that wrong and it doesn't make any difference.